src/databricks/labs/ucx/azure/access.py

```diff
@@ -21,16 +24,23 @@
 @dataclass
 class StoragePermissionMapping:
```

Comment:
@nfx : The change to this data class blew up the PR.

1. I will separate the PR to make it easier to comprehend and review.
2. I would like your input on if we can circumvent this change. Specifically the `role_name`, which is required to track the storage permission, so that we can set the appropriate permission level for the access connectors. The `StoragePermissionMapping` contains the Databricks privilege, however, this does not map to the Azure storage permissions. -> We could neglect the "Storage Blob Data Owner", give "Storage Blob Data Contributor" as highest permission as the access connectors should not be used to do POSIX access control